src/components/Wallet/WalletConfigInteractionButtons.jsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import { Button, Grid } from '@material-ui/core';
+
+import { CARAVAN_CONFIG } from './constants'
+
+const WalletConfigInteractionButtons = ({ onClearFn, onDownloadFn }) => {
```

Comment:
These were used in a couple places and felt cleaner to add as its own component, particularly now that it includes the sessionStorage management